server/app/views/applicant/ProgramIndexView.java

```diff
@@ -44,20 +39,17 @@ public final class ProgramIndexView extends BaseHtmlView {
   private final SettingsManifest settingsManifest;
   private final String authProviderName;
   private final ProgramCardViewRenderer programCardViewRenderer;
-  private final ZoneId zoneId;
 
   @Inject
   public ProgramIndexView(
       ApplicantLayout layout,
-      ZoneId zoneId,
+      ProgramCardViewRenderer programCardViewRenderer,
```

Comment:
question (non-blocking): Any reason to re-order this param?

Reply:
Ah no just churn while I was working with this code